src/vsphere_cpi/spec/integration/vm_host_rule_spec.rb

```diff
@@ -219,9 +247,9 @@
             expect(expected_host_group_mob.host.map(&:name)).to include(vm_host)
             vm_cluster_mob = vm.cluster == @cluster_name ? cluster_mob : second_cluster_mob
             expect(get_count_vm_host_affinity_rules(vm_cluster_mob)).to eq(1)
-            expect(find_vm_group_mob(expected_host_group_mob.name, vm_cluster_mob).vm).to include(vm.mob)
+            expect(find_vm_group_mob(expected_host_group_mob.name, clustermob: vm_cluster_mob).vm).to include(vm.mob)
             vm_host_rule = get_vm_host_affinity_rule(vm_cluster_mob)
-            expect(vm_host_rule.vm_group_name).to eq(expected_host_group_mob.name+VSphereCloud::Resources::Cluster::CLUSTER_VM_GROUP_SUFFIX)
+            expect(vm_host_rule.vm_group_name).to eq(expected_host_group_mob.name+default_drs_rule+VSphereCloud::Resources::Cluster::CLUSTER_VM_GROUP_SUFFIX)
```

Comment:
I think this line is too long. We might want a helper function for our specs. And also the addition rather than string interpolation makes this difficult to understand.